notifier/webhook/deliverer.go

```diff
@@ -96,9 +96,15 @@ func (d *Deliverer) Deliver(ctx context.Context, nID uuid.UUID) error {
 	}
 	buf := bytes.NewReader(b)
 
+	var headers http.Header = map[string][]string{}
+	headers["Content-Type"] = []string{"application/json"}
+	for key, val := range d.conf.Headers {
+		headers[key] = val
+	}
+
```

Comment:
I think this would be better written as:

```suggestion
	headers := d.conf.Headers.Clone()
	headers.Set("Content-Type", "application/json")

```

Reply:
What if `Headers` is `nil`?

Reply:
Hm yeah, that eats it because Clone returns nil. We could make Validate ensure it's non-nil, or not do it my suggested way. @ldelossa thoughts?

Reply:
I think doing this in Validate method makes sense. 

The webhook deliverer will *always* be sending json data. So in the "Validate()" method, merge any incoming headers with the "Content-type" header and construct the deliverer with the the merged http.Header map.